File: app/src/main/java/br/com/codeteam/ctanywhere/view/Emoji.java
package br.com.codeteam.ctanywhere.view;

/**
 * Created by Bruno Rodrigues e Rodrigues on 18/04/17.
 */

public class Emoji {
    
    public static String getEmojiByUnicode(int unicode){
        return new String(Character.toChars(unicode));
    }
}

File: app/src/main/java/br/com/codeteam/ctanywhere/view/Emoji.kt
package br.com.codeteam.ctanywhere.view

/**
 * Created by Bruno Rodrigues e Rodrigues on 18/04/17.
 */

object Emoji {

    fun getEmojiByUnicode(unicode: Int): String {
        return String(Character.toChars(unicode))
    }
}
